scst-scan/install-app-scanning.hbs.md

```diff
@@ -194,12 +220,17 @@ The following section describes how to configure service accounts and registry c
 
     Where:
 
-    - `imagePullSecrets.name` is the name of the secret used by the component to pull the scan component image from the registry.
-    - `secrets.name` is the name of the secret used by the component to publish the scan results.
+    - `imagePullSecrets.name` is the name of the secret used to pull the scan component image from the registry.
+    - `secrets.name` is the name of the secret used to publish the scan results.
+
+8. Apply the service account to your developer namespace by running:
+   ```console
+   kubectl apply -f publisher-sa.yaml
+   ```
 
 ## <a id="registry-retention-policy"></a> (Optional) Set up your registry retention policy
 
-If the registry specified to push scan results to support retention policies, you can configure the registry to delete old scan results automatically depending on your archival requirements. Scan result artifacts accumulate over time and with recurring scanning, artifacts can quickly consume storage space.
+If the registry specified to push scan results to supports retention policies, you can configure the registry to delete old scan results automatically depending on your archival requirements. Scan result artifacts accumulate over time and with recurring scanning, artifacts can quickly consume storage space.
```

Comment:
```suggestion
Although the scan artifacts are ingested in to the Tanzu Application Platform metadata store and information like packages and vulnerabilities parsed are stored, only a pointer to the the original SBOM location stored.  The original SBOM generated by the scan is not preserved within the metadata store.  VMware recommends these original architacts are kept according to your organizations archival requirements.  

Although not recommended, if the registry specified to push scan results to supports retention policies, you can configure the registry to delete old scan results automatically depending on your archival requirements. Scan result artifacts accumulate over time and can quickly consume storage space.
```
Couple thoughts here - we shouldn't recommend people to delete this as they have pointers stored to them within AMR, and it might be important to the customer to keep the original SBOM.  This will get more important as we move to signed sbom's/attestations.

Also, we shouldn't mention recurring scanning as it's not part of the product yet.